saleor/product/models.py

```diff
@@ -47,6 +48,8 @@ class Meta:
         verbose_name = pgettext_lazy('Category model', 'category')
         verbose_name_plural = pgettext_lazy('Category model', 'categories')
         app_label = 'product'
+        permissions = (('view_category', 'Can View Category in Dashboard'),
+                       ('edit_category', 'Can Edit Category in Dashboard'))
```

Comment:
What's the idea behind this title-like capitalization? Why have all permissions end in "in Dashboard"? Are we planning to introduce a second set of permissions for other use cases such as REST or GraphQL? If not the `is_staff` field is responsible for the "in dashboard" part.

Reply:
As in Django documentation. This second value has to be "Human readible". I can simplyfi this to just "Edit"/"View" or "edit"/"view".

Reply:
Sorry, I meant the Way It is Capitalized.

Reply:
ok i will turn it off